docs/master/api-reference/directives.md

```diff
@@ -1111,6 +1111,47 @@ class PlacedOrder
 }
 ```
 
+## @extends
+
+```graphql
+"""
+This allows types to extends other Types
+
+it inherits properties from the parent type. 
```

Comment:
What about directives on the parent type? Are those inherited as well?

Reply:
Yes. They are.

Reply:
This might be problematic with many directives, e.g. `@namespace`, `@interface` or `@enum`.